common/lib/wifi/src/lib.rs

```diff
@@ -51,8 +51,8 @@ pub fn wifi(
     };
 
     wifi.set_configuration(&Configuration::Client(ClientConfiguration {
-        ssid: ssid.into(),
-        password: pass.into(),
+        ssid: ssid.try_into().unwrap(),
```

Comment:
instead of unwraping maybe something like ".expect("Could not put given ssid into wifi config")," Currently the training is build with panic-imidiate-abort and a user will not see this unwrap.